demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -385,6 +390,9 @@ def create_incident(self, client: DefaultApi) -> Optional[Incident]:
         search_filter = demisto_client.demisto_api.SearchIncidentsData()
         inc_filter = demisto_client.demisto_api.IncidentFilter()
         inc_filter.query = f'id: {inc_id}'
+        if IS_XSIAM:
+            # in xsiam `create_incident` response don`t return created incident id.
+            inc_filter.query = f'name: "{incident_name}"'
```

Comment:
```suggestion
            inc_filter.query = f'name:"{incident_name}"'
```

Reply:
Why is it required? I copied it from `inc_filter.query = f'id: {inc_id}'`